vpr/src/route/connection_box_lookahead_map.h

```diff
@@ -2,16 +2,88 @@
 #define CONNECTION_BOX_LOOKAHEAD_H_
 
 #include <vector>
+#include <list>
 #include "physical_types.h"
 #include "router_lookahead.h"
+#include "router_lookahead_map_utils.h"
+#include "connection_box.h"
+
+template<typename T>
+class BoundingBox {
```

Comment:
BoundingBox should probably just be a `struct`, give that all of the members are public and all of the methods are public.